rpc-jobs/jjb-setup.yml

```diff
@@ -1,4 +1,24 @@
 - job:
+    name: 'Merge-Trigger-JJB'
+    project-type: freestyle
```

Comment:
I'd prefer to use pipeline but couldn't work out how to specify a branch for post merge trigger with a pipeline job.